addon/object/computed.js

```diff
@@ -1,10 +1,11 @@
 import Ember from 'ember';
-
 import {
   decoratedPropertyWithRequiredParams,
   decoratedPropertyWithOptionalCallback
 } from '../utils/decorator-macros';
 
+const { assert } = Ember;
```

Comment:
No worries. 😊

While we're at it, would you like another PR upgrading the rest of the code base to use module imports?